src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({
+                'is-active': tab.id === currentTab.id,
+              })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                onClick={() => {
+                  if (currentTab.id !== tab.id) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)